software/wireshark/en.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Wireshark"
+lastmod = "2018-08-10T19:35:05+02:00"
++++
+# Wireshark
+[Wireshark](https://www.wireshark.org/) is a widely-used network protocol analyzer that lets you see what’s happening on a network at a microscopic level.
```

Comment:
A newline is needed before the header and description. While this is valid Markdown, our Help Center indexer checks for a specific header format.